test/bundler-integration/token-paymaster/biconomy-token-paymaster-specs.ts

```diff
@@ -294,8 +290,14 @@ describe("Biconomy Token Paymaster (with Bundler)", function () {
 
       await environment.sendUserOperation(userOp, entryPoint.address);
 
-      const ev = await getUserOpEvent(entryPoint);
-      expect(ev.args.success).to.be.true;
+      // const ev = await getUserOpEvent(entryPoint);
+      // expect(ev.args.success).to.be.true;
```

Comment:
this can be removed

Reply:
need to discuss this with @ankurdubey521

Reply:
done. pushed updated code